R/class-workbook.R

```diff
@@ -1056,6 +1079,34 @@ wbWorkbook <- R6::R6Class(
       #   - Comments ???
       #   - Slicers
 
+      if (external_wb) {
+
+        # FIXME we copy all references from a workbook over to this workbook.
+        # This is not going to work, if multiple images from different
+        # workbooks are used. The references are called imageX.jpg and will
+        # overwrite each other. This needs a better solution
```

Comment:
maybe get the file path from drawings relationship, copy it into the worksheets media folder, make the path a unique string and replace it in drawings relationship